coral/src/app/features/requests/components/DeleteRequestDialog.tsx

```diff
@@ -3,6 +3,11 @@ import { Dialog } from "src/app/components/Dialog";
 type DeleteRequestModalProps = {
   deleteRequest: () => void;
   cancel: () => void;
+  // @TODO will be required later, will add it as optional
```

Comment:
I don't think it should be required :o Not every action dispatched by the actions in this dialog will be async, and requiring to pass `isLoading=false` in those sync cases seems cumbersome.

We may want to add `false` as default value to `isLoading`, though.  That could be done in this PR, I think ^^

```tsx
function DeleteRequestDialog({
  cancel,
  deleteRequest,
  isLoading = false,
}: DeleteRequestModalProps) {
```

Reply:
🤔  Isn't deleting always going to be async? I thought if since we're using this Modal always in this context where we know we want to handle the loading state, it would be safer to make it required (so we can't forget it! :D)